sources/packages/web/src/composables/useFormioDropdownLoader.ts

```diff
@@ -38,6 +43,46 @@ export function useFormioDropdownLoader() {
     );
   };
 
+  /**
+   * Populate program intensity dropdown.
+   * @param form form.
+   * @param isFulltimeAllowed is fulltime allowed.
+   * @param dropdownName dropdown name.
+   */
+  const loadProgramIntensityDetails = async (
+    form: any,
+    isFulltimeAllowed: boolean,
+    dropdownName: string,
+  ) => {
+    return loadDropdown(
+      form,
+      dropdownName,
+      getProgramIntensityDetails(isFulltimeAllowed),
+    );
+  };
+
+  const getProgramIntensityDetails = (isFulltimeAllowed: boolean) => {
```

Comment:
Method comments missing. In this case, unless reused, I do not see a need to create a separated method.

Reply:
`loadDropdown` requires the 3rd parameter to be a function. That is why created a separate function.